pkgs/by-name/ll/lldap-cli/package.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, bash
+, coreutils
+, gnugrep
+, gnused
+, jq
+, curl
+, makeWrapper
+}:
+stdenv.mkDerivation {
+  pname = "lldap-cli";
+  version = "nightly-2024-02-24";
+
+  src = fetchFromGitHub {
+    owner = "Zepmann";
+    repo = "lldap-cli";
+    rev = "d1fe50006c4a3a1796d4fb2d73d8c8dcfc875fd5";
+    hash = "sha256-ZKRTYdgtOfV7TgpaVKLhYrCttYvB/bUexMshmmF8NyY=";
+  };
+
+  buildInputs = [ bash coreutils gnugrep gnused jq curl ];
```

Comment:
```suggestion
```
these don't do anything as you're manually wrapping, and there's no hook here AFAICT that would use this attr

Reply:
Thanks a lot for your feedback. I'm new to this ecosystem and I'm very happy to improve my contributions over time.
This issue appeared, as I blindly followed the example for shell scripts in the wiki over [here](https://nixos.wiki/wiki/Shell_Scripts#Packaging).

Reply:
For future reference, the wiki is often out-of-date, though this is a good first PR.